### ISSUE TITLE
Error on list Computers

### ISSUE THREAD

**Issue by leirbagvier** [glpi-project/glpi#2466]
I'm using **FusionInventory** and creating computer items automatically in GLPI; I can see the complete Computer Item and its components, software, etc in plugin page;

But when I add a complete inventory can't list `Computer` screen anymore, in debug mode I see the following error:

```
Fatal error: Call to undefined function _() in /var/www/glpi/inc/networkportfiberchannel.class.php on line 331
```

**Comment by leirbagvier** [comment]
Based on similar situations, I found that the problem can be fixed just editing the `line 331` as follow:

Change this line:
```
$tab[62]['name']          = _('Network fiber outlet');
```

To this:
```
$tab[62]['name']          = __('Network fiber outlet');
```

**Comment by trasher** [comment]
This has been fixed already; see #2316 ;)
